src/pages/en/guides/integrations-guide/mdx.mdx

```diff
@@ -114,19 +115,48 @@ import rehypeMinifyHtml from 'rehype-minify-html';
 export default defineConfig({
   integrations: [
     mdx({
-      syntaxHighlight: 'shiki',
-      shikiConfig: { theme: 'dracula' },
-      remarkPlugins: [remarkToc],
-      rehypePlugins: [rehypeMinifyHtml],
-      remarkRehype: { footnoteLabel: 'Footnotes' },
-      gfm: false,
-    })
-  ]
-})
+      remarkPlugins: [exampleRemarkPlugin],
+    }),
+  ],
+});
+```
+
+…every MDX file will have `customProperty` in its frontmatter! See [our Markdown documentation](/en/guides/markdown-content/#example-injecting-frontmatter) for more usage instructions and a [reading time plugin example](/en/guides/markdown-content/#example-calculate-reading-time).
```

Comment:
```suggestion
…every MDX file will have `customProperty` in its frontmatter! See [our Markdown documentation](/en/guides/markdown-content/#modifying-frontmatter-programmatically) for more usage instructions and a [reading time plugin example](/en/guides/markdown-content/#example-calculate-reading-time).
```

Reply:
updating and outdated link manually. Assuming this is the only problem with these files, I'll go back and update the MDX guide at the source so that these can all merge.